src/quo2/components/notifications/toast/view.cljs

```diff
@@ -38,42 +45,47 @@
      :blur-type     :transparent
      :overlay-color :transparent}]
 
-   [rn/view {:style (style/content-container override-theme)}
+   [rn/view {:style (style/content-container theme)}
     [rn/view {:style style/left-side-container}
      left]
     [rn/view {:style style/right-side-container}
      (when title
        [text/text
         {:size                :paragraph-1
          :weight              :semi-bold
-         :style               (style/title override-theme)
+         :style               (style/title theme)
          :accessibility-label :toast-title}
         title])
      (when text
        [text/text
         {:size                :paragraph-2
          :weight              :medium
-         :style               (style/text override-theme)
+         :style               (style/text theme)
          :accessibility-label :toast-content}
         text])]
     right]])
 
+(def ^:private toast-container (quo.theme/with-theme toast-container-internal))
+
 (defn toast
   [{:keys [icon icon-color title text action undo-duration undo-on-press container-style
-           override-theme user]}]
-  [toast-container
-   {:left            (cond icon
-                           [icon/icon icon
-                            (cond-> (style/icon override-theme)
-                              icon-color
-                              (assoc :color icon-color))]
+           theme user]}]
+  (let [context-theme (or theme (quo.theme/get-theme))]
```

Comment:
is this a temporary solution while we update uses across the codebase? 🤔

Reply:
No, That's for the theme provider below that line.
Fallback to user-theme if a theme is not specified.

It's similar to what we do with screens or bottom sheets.
https://github.com/status-im/status-mobile/blob/33d41dc5d55b6e976edaf23ac2caf6ab3d1c87bc/src/status_im2/navigation/view.cljs#L66

Reply:
anyway it's beyond this pr 👍